src/components/dataset/query/JadeDropdown.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+
+const styles = () => ({
+  root: {
+    width: '100%',
+  },
+});
+
+export class JadeDropdown extends React.PureComponent {
+  static propTypes = {
+    classes: PropTypes.object,
+    name: PropTypes.string,
+    onSelectedItem: PropTypes.func,
+    options: PropTypes.array,
+    value: PropTypes.string,
+  };
+
+  render() {
+    const { classes, options, onSelectedItem, value, name } = this.props;
+
+    return (
+      <form autoComplete="off">
```

Comment:
It's not clear to me what this is rendering. Could you comment on this?

Reply:
It renders a dropdown component, like the "Select Asset" box in the gif. I added a comment to the render method!